Language/Variables/Implicitly_Induced_Getters_and_Setters/definition_A07_t01.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Case ⟨Setter: Late-final variable with no declared type, no
+/// initialization⟩. A variable declaration of the form static? late final id;
+/// implicitly induces a setter with the header void set id(dynamic x). An
+/// execution of said setter in a situation where the variable id has not been
+/// bound will bind id to the object that the argument x is bound to. An
+/// execution of the setter in a situation where the variable id has been bound
+/// to an object will incur a dynamic error
+///
+/// @description Checks that the static type of an implicit setter of a late
+/// final variable with no declared type and no initialization is `dynamic`
```

Comment:
How would we be able to test that? If you want to distinguish a `void Function(dynamic)` from a `void Function(Object?)` then you can use a tear-off, `runtimeType`, and test equality with `typedef F1 = void Function(dynamic);` respectively `typedef F2 = void Function(Object?);` (those function types must be non-equal because type equality makes a distinction between `dynamic` and `Object?`). But you can't tear off a setter...

Reply:
Deleted this test

Reply:
OK.